src/interactive/AccordionPanel.jsx

```diff
@@ -66,15 +66,34 @@ class AccordionPanel extends React.Component {
 	 *
 	 * Updates state to toggle `AccordionPanel` open and closed
 	 */
-	componentWillUpdate(nextProps, nextState) {
+	componentWillReceiveProps(nextProps) {
+		console.log('using componentWillReceiveProps which should be better');
 		if (nextProps.isOpen !== this.state.isOpen) {
+			const isOpen = nextProps.isOpen;
 			this.setState({
-				isOpen: nextProps.isOpen,
-				height: this.getHeight(nextProps.isOpen)
+				isOpen: isOpen,
+				height: this.getHeight(isOpen)
 			});
 		}
 	}
```

Comment:
I read that you shouldn't update state in `componentWillUpdate`, so using `componentWillReceiveProps`